README.md

```diff
@@ -52,34 +52,36 @@ Repository commands
 You can run several commands on source files. They are available as yarn
 subcommands:
 
-- `yarn build` — build, transpile, pack
-- `yarn dev-browser` — run dev-version of browser IDE on localhost
-- `yarn electron-dist` — build OS-specific distributive
+- `yarn build` — build, transpile, pack all
+- `yarn build:electron` — build desktop IDE only
+- `yarn build:cli` — build CLI tools only
+- `yarn dev:browser` — run dev-version of browser IDE on localhost
+- `yarn dist:electron` — build OS-specific distributive of desktop IDE
 - `yarn test` — run unit tests
 - `yarn test:watch` — like test, but watches for changes with auto-retest
```

Comment:
`test:watch` does not conform to new naming scheme

Reply:
Good catch. Do you have a suggestion? `yarn test-watch`?

Reply:
Sounds good 👍🏻
Or we could just get rid of it. It's set up only for `xod-project` and `xod-client` anyway

Reply:
OK. The question is do we use it? I’m not. @evgenykochetkov @brusherru do you have the development pattern with tests running on file change?

Reply:
I'm not using it.

Reply:
I'm not using it too :)

Reply:
OK, removed it.